games.js

```diff
@@ -0,0 +1,74 @@
+var GAMES = [
+    {
+        "code": "the-heist",
```

Comment:
can we call this path or something similar? 'code' sounds like the "gameCode" that we are using to identify the games in the server, and the naming is not consistent with this, so I think it'd be confusing